modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsKeywordReader.scala

```diff
@@ -191,7 +191,8 @@ trait TcsKeywordDefaults {
 }
 
 object DummyTargetKeywordsReader {
-  def apply[F[_]: Applicative]: TargetKeywordsReader[F] = new TargetKeywordsReader[F] {
+
+  class DummyTargetKeywordsReaderImpl[F[_]: Applicative] extends TargetKeywordsReader[F] {
```

Comment:
I think this can be a `private` class

Reply:
I use it in the unit tests, I need it accessible from there.